public/index.html

```diff
@@ -32,7 +32,8 @@
       })();
     </script>
     <script>
-      (function (env) {
+      (function () {
```

Comment:
are we still using this file? for development perhaps?

Reply:
This file is loaded before the javascript kicks in.

Reply:
This file is not used. The JS does not have to kick in, the site is statically generated. Next.js creates ints own index file unless the server ins misconfigured.